src/utils/ErrorUtils.js

```diff
@@ -0,0 +1,60 @@
+//@flow
+import ErrorCodes from './ErrorCodes';
+import TCATUtils from './TCATUtils';
+
+function getErrorCode(message) {
+
+    message = message.toLowerCase();
+
+    if (message.includes('is out of bounds')) {
+        return ErrorCodes.OUT_OF_BOUNDS; //Point 0 is out of bounds: 42.4929015990742,-76.9723900315137
```

Comment:
All these comments should have a space after the `//`, for example:

`// Point 0 is out of bounds:...`